src/connection/xmpp/Connector.ts

```diff
@@ -50,6 +52,29 @@ export default class Connector {
       }
    }
 
+   public isWebsocketUrl(url?: string)
+   {
+        if (/^ws?:/.test(url?url:this.connectionParameters.get('url'))||/^wss?:/.test(url?url:this.connectionParameters.get('url')))
```

Comment:
see `/^wss?:/.test(connectionArgs[1])`

Reply:
changed... could not reproduce why i did it..